### PR TITLE
Migration to Convox, multi language support, improved caching

### PR DESCRIPTION
**Pull request by wojtekk**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+app.use((req, res) => {
+  const rootDomain = req.hostname
+    .split('.')
+    .reverse()
+    .slice(0, 2)
+    .reverse()
+    .join('.')
+  switch (rootDomain) {
+    case 'whatisneeded.pl':
+      res.sendFile(`${buildDir}/index_en.html`, { maxAge: 300 })
```
Comment (wojtekk) [inline index.js:24]: @sjedlikowski WDYT? For HTMLs

File: index.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+const express = require('express')
+const app = express()
+const PORT = process.env.PORT || 3000
+
+const buildDir = `${__dirname}/build`
+
+app.use(
+  express.static(buildDir, { maxAge: 2629746, immutable: true, index: false })
```
Comment (wojtekk) [inline index.js:8]: @sjedlikowski WDYT? For Static files (1 month)?

File: index.js (hunk 1 above)
Comment (sjedlikowski) [reply index.js:24]: Yeah, 5 minutes should be fine.

File: index.js (hunk 2 above)
Comment (sjedlikowski) [reply index.js:8]: Yup, 1 month will be fine as well

**Review by sjedlikowski — APPROVED** [review]

**Pull request merged**